src/linear_algebra/matrix.lean

```diff
@@ -186,10 +186,12 @@ linear_equiv.trans (linear_equiv.arrow_congr (equiv_fun_basis hv₁) (equiv_fun_
 end linear_equiv_matrix
 
 namespace matrix
+open_locale matrix
 
 section trace
 
-variables {R : Type v} {M : Type w} [ring R] [add_comm_group M] [module R M]
+variables (m n)
+variables (R : Type v) (M : Type w) [ring R] [add_comm_group M] [module R M]
```

Comment:
We only want explicit types for the definitions of linear maps. There's no need for these to be explicit for the theorems.

Reply:
I see, thanks.